src/overlays/actors/ovl_En_Ru/z_en_ru.h

```diff
@@ -10,7 +13,28 @@ typedef void (*EnRuActionFunc)(struct EnRu*, PlayState*);
 typedef struct EnRu {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnRuActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x30C];
+    /* 0x148 */ SkelAnime skelAnime;
+    /* 0x18C */ ColliderCylinder collider;
+    /* 0x1D8 */ UNK_TYPE1 pad1D8[0x4];
+    /* 0x1DC */ Path* path; // path is prepared in EnRu_Init, but unused after
+    /* 0x1E0 */ UNK_TYPE1 pad1E0[0x4];
+    /* 0x1E4 */ Vec3f leftFootPos;
+    /* 0x1F0 */ Vec3f rightFootPos;
+    /* 0x1FC */ u8 isLeftFootGrounded;
+    /* 0x1FD */ u8 isRightFootGrounded;
+    /* 0x1FE */ Vec3s jointTable[RU2_LIMB_MAX];
+    /* 0x288 */ Vec3s morphTable[RU2_LIMB_MAX];
+    /* 0x312 */ Vec3s trackTarget;
+    /* 0x318 */ Vec3s headRot;
+    /* 0x31E */ Vec3s torsoRot;
+    /* 0x324 */ UNK_TYPE1 pad324[0x12];
+    /* 0x336 */ s16 limbRotTableY[RU2_LIMB_MAX];
+    /* 0x364 */ s16 limbRotTableZ[RU2_LIMB_MAX];
+    /* 0x392 */ UNK_TYPE1 pad392[0x2];
+    /* 0x394 */ Vec3f bodyPartsPos[0xF]; // ARRAY_SIZE(sRuBodyParts)
```

Comment:
Should be in dec since it's a known array (of limb-related things). And then you can declare `sRuBodyParts` with the array count of this to force them sync'd.

Reply:
Uhh... Dont think this works
```
static s8 sRuBodyParts[ARRAY_SIZE(this->bodyPartsPos)] = {
    0, 0, 0, 0, 3, 4, 0, 6, 7, 0, 9, 10, 0, 12, 13,
};  
``` as `this` doesn't exist in this scope

Reply:
Will have to be a define, then.